alphadia/outputtransform.py

```diff
@@ -981,15 +977,22 @@ def _build_run_stat_df(
             optimization_manager = manager.OptimizationManager(
                 path=optimization_manager_path
             )
-            optimization_stats[MS2_ERROR] = optimization_manager.ms2_error
-            optimization_stats[MS1_ERROR] = optimization_manager.ms1_error
-            optimization_stats["rt_error"] = optimization_manager.rt_error
-            optimization_stats["mobility_error"] = optimization_manager.mobility_error
+            optimization_stats[OutputKeys.MS2_ERROR] = optimization_manager.ms2_error
```

Comment:
@GeorgWa I'd suggest using this idiom for all string constants short- to mid-term